### DISCUSSION TITLE
Add a field for gift status in Preparation

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5665] [Feature Requests]
> Is there a way we can add a field to the form that gives users an at a glance view of if there is a gift associated with a preparation? - e.g. equivelant of the isOnLoan?

**Requested By:** [RBGE on the Community Forum](https://discourse.specifysoftware.org/t/finding-interations-associated-with-a-preparation/864/5?u=specify)

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/finding-interations-associated-with-a-preparation/864/6

**Comment by g1rly-c0d3r** [comment]
related to https://github.com/specify/specify7/issues/8174#issue-4625879385
